### DISCUSSION TITLE
unable to connect to youtube on firefox

### DISCUSSION THREAD

**Discussion by jdog124** [atauenis/webone#146] [Q&A] [answered]
when i try to connect to youtube on firefox 4 using the wobone proxy it gives me a certificate error, does anyone have a solution to this issue? I am new to using proxy's

**Comment by atauenis** [comment]
You need to import proxy's root certificate to Firefox's certificate storage.

**Reply by jdog124** [reply]
okay, thanks but also how to i do that

**Reply by atauenis** [reply, marked as answer]
It's easy. Open http://proxy:8080/ and then click "WebOne CA Root Certificate" link. Then import the certificate to Firefox database.

Detailed instructions: https://github.com/atauenis/webone/wiki/Installing-the-Root-Certificate
